pages/privacy.js

```diff
@@ -0,0 +1,254 @@
+        </PrivacyTextBlock>
+        <Divider />
+        <FAQHeading>Legal Basis for Processing Personal Data Under General Data Protection Regulation (GDPR)</FAQHeading>
+        <PrivacyTextBlock>
+        If you are from the European Economic Area (EEA), Flossbank’s legal basis for collecting and using the personal information described in this Privacy Policy depends on the Personal Data we collect and the specific context in which we collect it.
+        </PrivacyTextBlock>
+        <PrivacyTextBlock>
+        Flossbank may process your Personal Data because:
+        </PrivacyTextBlock>
+        <UnorderedList>
+          <ListItem>
+          We need to perform a contract with you
+          </ListItem>
+          <ListItem>
+          You have given us permission to do so
+          </ListItem>
+          <ListItem>
+          For payment processing purposes
+          </ListItem>
+          <ListItem>
+          To comply with the law
+          </ListItem>
+        </UnorderedList>
+        <Divider />
+        <FAQHeading>Your Data Protection Rights Under General Data Protection Regulation (GDPR)</FAQHeading>
+        <PrivacyTextBlock>If you are a resident of the European Economic Area (EEA), you have certain data protection rights. Flossbank aims to take reasonable steps to allow you to correct, amend, delete, or limit the use of your Personal Data.</PrivacyTextBlock>
+        <PrivacyTextBlock>If you wish to be informed what Personal Data we hold about you and if you want it to be removed from our systems, please contact us.</PrivacyTextBlock>
+        <PrivacyTextBlock>In certain circumstances, you have the following data protection rights:</PrivacyTextBlock>
+        <UnorderedList>
+          <ListItem>
+          The right to access, update or to delete the information we have on you.
+          </ListItem>
+          <ListItem>
+          The right of rectification. You have the right to have your information rectified if that information is inaccurate or incomplete.
+          </ListItem>
+          <ListItem>
+          The right to object. You have the right to object to our processing of your Personal Data.
+          </ListItem>
+          <ListItem>
+          The right of restriction. You have the right to request that we restrict the processing of your personal information.
+          </ListItem>
+          <ListItem>
+          The right to data portability. You have the right to be provided with a copy of your Personal Data in a structured, machine-readable and commonly used format.
+          </ListItem>
+          <ListItem>
+          The right to withdraw consent. You also have the right to withdraw your consent at any time where Flossbank relied on your consent to process your personal information.
```

Comment:
This implies there was consent to begin with... so we will need to add something on the sign up page like a check box or text that says they are consenting. 

Also because we don't use cookies before sign up, I think (hoping) we can avoid the GDPR pop up on the splash page and instead have GDPR consent baked into the sign up page or something.

Reply:
I think this line is saying "if we rely on your consent to process any personal information, you can always revoke that consent"

we should also have a checkbox that points them to the privacy policy i think

Reply:
can you include the checkbox in this PR

Reply:
done